documentation/modules/overview/con-configuration-points-connect.adoc

```diff
@@ -56,3 +56,45 @@ To create the image, you can use:
 
 * A Kafka container image on {DockerRepository} as a base image
 * OpenShift {docs-okd} and the {docs-okd-s2i} framework to create new container images
+
+[discrete]
+== Managing connectors
+
+You can use the `KafkaConnector` resource or the link:https://kafka.apache.org/documentation/#connect_rest[Kafka Connect API] to create and manage connector instances in a Kafka Connect cluster.
+The `KafkaConnector` resource offers a Kubernetes-native approach, and is managed by the Cluster Operator.
+
+The `spec` for the `KafkaConnector` resource specifies the connector class and configuration settings, as well as the maximum number of connector _tasks_ to handle the data.
+
+[discrete]
+== Example YAML showing KafkaConnector configuration
+[source,yaml,subs="attributes+"]
+----
+apiVersion: kafka.strimzi.io/v1alpha1
```

Comment:
`{KafkaApiVersion}` attribute as below?

(Not sure I've been using this attribute consistently either.)